public_desktop/desktop-app.js

```diff
@@ -92,6 +94,42 @@ function startDesktopApp() {
 
 	debug = gGebug( 'desktop:browser' );
 
+	try {
+		var buildEditorContextMenu = electron.remote.require( '../desktop/lib/menu/editor-context-menu' );
+		var buildGeneralContextMenu = electron.remote.require( '../desktop/lib/menu/general-context-menu' );
+	} catch (e) {
+		debug( "Error loading context menus", e.message);
+		loadContextMenu = false;
+	}
+	function contextMenu( ev ) {
```

Comment:
Can we move the functions up along with the others? They break the flow of the code which can be a bit confusing.